app/portainer/components/stack-duplication-form/stack-duplication-form-controller.js

```diff
@@ -18,13 +18,18 @@ angular.module('portainer.app').controller('StackDuplicationFormController', [
     ctrl.duplicateStack = duplicateStack;
     ctrl.migrateStack = migrateStack;
     ctrl.isMigrationButtonDisabled = isMigrationButtonDisabled;
+    ctrl.stackNameAvailable = true;
+
+    ctrl.stackNameChange = function(name) {
```

Comment:
Please rename the function and use the following pattern when associating functions to this controller:

```javascript

ctrl.onStackNameChange = onStackNameChange;

function onStackNameChange(name) {
...
}
```